vehicle-detection/darkflow/net/ops/convolution.py

```diff
@@ -85,8 +85,9 @@ def batchnorm(self, layer, inp):
                 'updates_collections' : None,
                 'is_training': layer.h['is_training']
                 })
-            v = tf.__version__.split('.')[1]
-            if int(v) < 12: key = 'initializers'
+            v_ma = tf.__version__.split('.')[0]
+            v_mi = tf.__version__.split('.')[1]
+            if int(v_ma) < 1 & int(v_mi) < 12: key = 'initializers'
```

Comment:
@kiranvdhotre Please pay attention to the variable names. One is for the version's major value and the other for the minor value, **v_ma** and **v_mi** respectively.